lib/game_runner.rb

```diff
@@ -20,13 +21,18 @@ def grid_printer
     @grid_printer ||= GridPrinter.new(stdout, grid)
   end
 
-  def player_input
-    @player_input ||= PlayerInput.new(stdin, grid, printer)
+  def player
+    @player ||= Player.new(stdin, grid, printer)
+  end
+
+  def computer_player
+    @computer_player ||= ComputerPlayer.new(grid)
   end
 
   def run
     introduce_game
-    player_1_turn
+    player_turn
```

Comment:
This is very cool that it boils down to two similar statements. 

What would it take to have it be this instead

```ruby
player.take_turn
computer_player.take_turn
```

A lot of stuff would need to move to those player classes. Also, I wonder about the players maintaining their own markers!